~

```diff
@@ -0,0 +1,7 @@
+Merge branch 'master' of https://github.com/munkhuushmgl/java-mediatranslation
```

Comment:
Samples look fine.

Nit: not sure how this file got added, but if you can remove great, if not, we can figure it out later.

Reply:
Just removed it!

Thanks a lot for quick reply and review.